### DISCUSSION TITLE
Time schedule for motion detection to save space.

### DISCUSSION THREAD

**Discussion by UbuntuRikard** [Motion-Project/motion#1624] [General]
A feature request
A time schedule where motion is not detection movment.
As if I only want motion detection in the night.
Or to watch a bird place.

**Comment by olddog55** [comment]
Easy to implement on your own.  No enhancement required.
Traditionally, in cron, use /etc/init.d/motion [start|stop]
If your os uses systemd, see https://linuxconfig.org/how-to-schedule-tasks-with-systemd-timers-in-linux

**Comment by UbuntuRikard** [comment]
Thank's.
Sometime you don't see what is obvious.
I know the crontab, not the other.
